<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerZoomModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerZoomModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="zoom3DCollapsibleBtn" native="true">
     <property name="text" stdset="0">
      <string>3D view aspect ratio</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="viewNode3DHLayout">
        <item>
         <widget class="QLabel" name="viewNode3DLabled">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>241</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>View:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="viewNode3DSelector">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>0</height>
           </size>
          </property>
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLViewNode</string>
           </stringlist>
          </property>
          <property name="showHidden" stdset="0">
           <bool>true</bool>
          </property>
          <property name="addEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="removeEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="renameEnabled" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="xZoom3DHLayout">
        <item>
         <widget class="QLabel" name="xZoom3DLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>zoom-X:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qSlicerGainWidget" name="xGain3DWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="yZoom3DHLayout">
        <item>
         <widget class="QLabel" name="yGain3DLabel">
          <property name="text">
           <string>zoom-Y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qSlicerGainWidget" name="yGain3DWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="zZoom3DHLayout">
        <item>
         <widget class="QLabel" name="zGain3DLabel">
          <property name="text">
           <string>zoom-Z:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qSlicerGainWidget" name="zGain3DWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="zoom2DCollapsibleBtn" native="true">
     <property name="text" stdset="0">
      <string>2D slice aspect ratio</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="viewNode2DHLayout">
        <item>
         <widget class="QLabel" name="sliceNode2DLabel">
          <property name="text">
           <string>Slice:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="sliceNode2DSelector">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLSliceNode</string>
           </stringlist>
          </property>
          <property name="showHidden" stdset="0">
           <bool>true</bool>
          </property>
          <property name="addEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="removeEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="renameEnabled" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="zoom2DFormLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="xZoom2DLabel">
          <property name="text">
           <string>horizontal zoom:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="qSlicerGainWidget" name="xGain2DWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="yZoom2DLabel">
          <property name="text">
           <string>vertical zoom:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="qSlicerGainWidget" name="yGain2DWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerGainWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerGainWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QComboBox</extends>
   <header>qMRMLNodeComboBox.h</header>
   <slots>
    <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
    <slot>setMRMLScene(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerZoomModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>viewNode3DSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>4</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerZoomModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>sliceNode2DSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>4</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
  <slot>setMRMLScene(vtkMRMLScene*)</slot>
 </slots>
</ui>
